### ISSUE TITLE
Some windows tests fail with slighly different numerical results

### ISSUE THREAD

**Issue by gbaraldi** [JuliaLang/julia#48101]
The "new" AWS machines we are renting have Ivy Bridge CPUs, which do not have fma, and they lead to small differences.
@oscardssmith Do you know if that's expected or do we have something to fix here?

**Comment by DilumAluthge** [comment]
An example failure:

```
Expression: sinpi(5 // 6) == 0.5
   Evaluated: 0.49999999999999994 == 0.5
```

**Comment by oscardssmith** [comment]
the lack of fma strikes again.

**Comment by DilumAluthge** [comment]
Can we somehow detect whether the machine supports fma? And if it does, test for equality, but if it does not, test for approximate equality?

**Comment by KristofferC** [comment]
> Can we somehow detect whether the machine supports fma?

`Core.Compiler.have_fma`, presumably.

**Comment by giordano** [comment]
Presumably fixed by #48102?  We don't have those agents anymore to verify it.
